core/services/relay/evm/chain_reader_test.go

```diff
@@ -219,3 +234,29 @@ func (h *helper) MaxWaitTimeForEvents() time.Duration {
 
 	return maxWaitTime
 }
+
+func (h *helper) TXM(t *testing.T, client client.Client, db *sqlx.DB) evmtxmgr.TxManager {
```

Comment:
I don't really understand what is the point of adding CW into EVM CR tester? CW should be tested separately from CR